pkgs/by-name/pi/picom-pijulius/package.nix

```
@@ -0,0 +1,21 @@
+{ picom, lib, fetchFromGitHub, pcre }:
+picom.overrideAttrs (finalAttrs: previousAttrs: {
+  pname = "picom-pijulius";
+  version = "2024-05-17-unstable";
+
+  src = fetchFromGitHub {
+    owner = "pijulius";
+    repo = "picom";
+    rev = "e7b14886ae644aaa657383f7c4f44be7797fd5f6";
+    sha256 = "1xi7bk8yicdvpn17cjws9q4xff8dcgl664i6pkdyhfwwg3j6j1b1";
+    fetchSubmodules = true;
```

Nit: this repo doesn't use submodules, so fetchSubmodules can be removed (yes there is a .gitmodules file, but it's been empty for 5 years).
Also prefer `hash`. 
```suggestion
    hash = "sha256-YQVp5HicO+jbvCYSY+hjDTnXCU6aS3aCvbux6NFcJ/Y=";
```

---

Interesting, I thought that it was needed, as the instructions said so. Removing it